src/tests/JIT/opt/OSR/largefuncletframe.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <DebugType />
+    <Optimize>True</Optimize>
+   <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildProjectName).cs" />
+  </ItemGroup>
+  <PropertyGroup>
+    <CLRTestBatchPreCommands><![CDATA[
+$(CLRTestBatchPreCommands)
+set COMPlus_TieredCompilation=1
+set COMPlus_TC_QuickJitForLoops=1
+set COMPlus_TC_OnStackReplacement=1
+set COMPlus_OSR_HitLimit=2
+set COMPlus_JitRandomOnStackReplacement=15
+set COMPlus_JitStress=2
+]]></CLRTestBatchPreCommands>
+    <BashCLRTestPreCommands><![CDATA[
+$(BashCLRTestPreCommands)
+export COMPlus_TieredCompilation=1
+export COMPlus_TC_QuickJitForLoops=1
+export COMPlus_TC_OnStackReplacement=1
+export COMPlus_OSR_HitLimit=2
+export COMPlus_JitRandomOnStackReplacement=15
+export COMPlus_JitStress=2
+]]></BashCLRTestPreCommands>
```

Comment:
Is this going to be the first Pri-0 OSR test case?

Reply:
All the tests in this directory are OSR tests. This will be the first that relies on "Random" OSR and aggressive OSR transitions. Normally those are only tested in the experimental CI legs.

I could probably rewrite the test to trigger with default OSR settings, if you think that's preferable.

Reply:
Hmmm, "random" doesn't seem great in terms of repeatability? But if the seed is the method hash, then it's not actually random? Seems ok.